modules/bbp-simulation/src/main/resources/schemas/bbp/simulation/detailedcircuit/v0.1.0.json

```diff
@@ -18,31 +18,31 @@
       "nodeKind": "sh:BlankNodeOrIRI",
       "and": [
         {
-          "node": "{{base}}/schemas/hbphackathon/simulation/modelinstance/v0.1.0/shapes/ModelInstanceShape"
+          "node": "{{base}}/schemas/bbp/simulation/modelinstance/v0.1.0/shapes/ModelInstanceShape"
         },
         {
           "property": [
             {
               "path": "nsg:nodeCollection",
               "name": "Node collection",
               "description": "Node collection entity.",
-              "seeAlso": "{{base}}/schemas/hbphackathon/simulation/nodecollection/v0.1.0/shapes/NodeCollectionShape",
+              "seeAlso": "{{base}}/schemas/bbp/simulation/nodecollection/v0.1.0/shapes/NodeCollectionShape",
               "class": "nsg:NodeCollection",
               "minCount": 1
             },
             {
               "path": "nsg:edgeCollection",
               "name": "nrnPath",
               "description": "Location of nrn synapse file and additional circuit description files: start.ncs and start.target",
-              "seeAlso": "{{base}}/schemas/hbphackathon/simulation/edgecollection/v0.1.0/shapes/EdgeCollectionShape",
+              "seeAlso": "{{base}}/schemas/bbp/simulation/edgecollection/v0.1.0/shapes/EdgeCollectionShape",
               "class": "nsg:EdgeCollection"
             },
             {
               "path": "nsg:target",
               "name": "TargetFile",
               "description": "Optional parameter giving location of predefined targets stored in the named file",
               "editorialNote": "constrain with application/bbp-target",
-              "seeAlso": "{{base}}/schemas/hbphackathon/simulation/target/v0.1.0/shapes/TargetShape",
+              "seeAlso": "{{base}}/schemas/bbp/simulation/target/v0.1.0/shapes/TargetShape",
```

Comment:
We are missing TargetShape if all entities now will have distribution then it can just extend the EntityShape.